tox.ini

```diff
@@ -24,3 +31,13 @@ commands = {[marionette]commands}
 skip_install = true
 deps = -rrequirements/flake8.txt
 commands = flake8 {posargs:.}
+
+[flake8]
+ignore = E501
+
+[pytest]
+addopts = --develop -n=auto --verbose -r=a --driver=Firefox
```

Comment:
Got rid of it - answer: nothing, apparently :-)

See my earlier comment about the couple of doc pages I found and stabbed in the dark with, trying to solve the python setup.py develop issue.